assets/js/search.js

```diff
@@ -23,8 +23,7 @@ limitations under the License.
           }
 
           var query = $(this).val();
-          var searchPage = "{{ "docs/search/" | absURL }}?q=" + query;
-          document.location = searchPage;
+          document.location = "{{ "search/" | absURL }}?q=" + query;
```

Comment:
At the moment, the source is the same as found in the theme. The file could be removed.